content/docs/introducing-jsx.md

```diff
@@ -1,36 +1,36 @@
 ---
 id: introducing-jsx
-title: Introducing JSX
-permalink: docs/introducing-jsx.html
+title: Pagpapakilala sa JSX
+permalink: docs/pagpapakilala-sa-jsx.html
 prev: hello-world.html
 next: rendering-elements.html
 ---
 
-Consider this variable declaration:
+Mangyaring tingnan ang variable declaration na ito:
 
 ```js
 const element = <h1>Hello, world!</h1>;
 ```
 
-This funny tag syntax is neither a string nor HTML.
+Ang syntax na ito ay hindi isang string o HTML.
 
-It is called JSX, and it is a syntax extension to JavaScript. We recommend using it with React to describe what the UI should look like. JSX may remind you of a template language, but it comes with the full power of JavaScript.
+Ito ay tinatawag na JSX, at isa itong halimbawa ng syntax extension sa JavaScript. Inirerekomenda naming gamitin ito kasama ng React para bigyan ng porma ang UI. Maihahalintulad ang JSX sa isang template language, ngunit maaari rin itong ipaghalo sa iba pang JavaScript na code.
```

Comment:
- Inirerekomenda = Ipinapayo
- JSX may remind you of a template language = `Ang JSX ay magpapaalala sa inyo ng template language`
- but it comes with the full power of JavaScript = `pero dala nito ang isang daan porsyentong lakas ng JavaScript.` or `pero dala nito ang buong bisa at lakas ng JavaScript.`

Reply:
I don't want to translate word-per-word tho, so I guess Tagalog could use its own character regarding this.